### PR TITLE
#3 Create Provider and Recipient Content

### PR DESCRIPTION
**Pull request by darnjo**
Recreating this PR as the branch was renamed to use a "semantic" format. 

The previous PR (#2) was closed, so the review comments are no longer on this branch, [but can be found here](https://github.com/RESOStandards/reso-certification-help/pull/2/files).

### CODE REVIEW

File: docs/providers.md [hunk 1]
```diff
@@ -7,98 +7,147 @@ After logging in to [certification.reso.org](certification.reso.org), the endors
...
+_[Web API Report]_
+
 # Notifications
 Endorsements must be reviewed and approved by customers before they are certified. The review requests are sent via email.
 
-Sending notification of the Data Dictionary report will send also send the Web API Core report together in a bundled review notification where the recipient will be able to review and approve both reports from a single link.
+Selecting **Notify** for either Data Dictionary or the Web API Core report will send a bundled review notification for both endorsements. The recipient will be able to review and approve both reports from a single link that directs them to the Certification Summary page.
 
-**Note:** the only time the bundled notification is available is when BOTH Data Dictionary and Web API are in Passed status. Otherwise, the email will only be for the endorsement selected.
+**Note:** The bundled notification is available ONLY when BOTH Data Dictionary and Web API are in **Passed** status. Otherwise, the email will only include the single endorsement that was selected for notification.
```
Comment (Jad212s) [inline docs/providers.md:131]: Rewritten

File: docs/recipients.md [hunk 2]
```diff
@@ -24,48 +29,64 @@ The Data Elements view shows your field and lookup counts and how they compare t
...
 
 **Payload Size** - a measure of how much data was retrieved when sampling the selected resource. When All is selected, it's the average across all resources found, measured in megabytes (MB).
 
 **Response Time** - shows how long, on average, a request to the given server took to complete, measured in seconds(s).
 
-**Bandwidth** - is payload size divided by response time. Similar to an internet speed test, this metric shows how quickly consumers might expect to retrieve data from the given server, measured in megabytes per second (MB/s).
+**Seconds per 1K Records** - an estimate of how many seconds it might take to pull 1,000 records using pages of 100 records. It's based on the average response time found during data sampling and validation. Measured in seconds (s).
 
-**N/A** means that the given recipient has not completed Data Dictionary 1.7 payloads testing yet. Please contact RESO at certification@reso.org if you have any questions.
+**N/A** means that the given recipient has not completed Data Dictionary Availability 1.7 testing yet. Please contact RESO at certification@reso.org if you have any questions.
```
Comment (Jad212s) [inline docs/recipients.md]: Lowercase "availability" and move behind "1.7".

File: docs/recipients.md [hunk 3]
```diff
@@ -24,48 +29,64 @@ The Data Elements view shows your field and lookup counts and how they compare t
...
+
 The Performance View shows the data provider’s server metrics.
 
+_[Performance View Chart]_
+
 Performance is measured on:
 
 **Payload Size** - a measure of how much data was retrieved when sampling the selected resource. When All is selected, it's the average across all resources found, measured in megabytes (MB).
 
 **Response Time** - shows how long, on average, a request to the given server took to complete, measured in seconds(s).
```
Comment (Jad212s) [inline docs/recipients.md:46]: Space before (s).

Comment (Jad212s) [inline docs/recipients.md]: Space before (s).

File: docs/recipients.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
 # Notifications
-You will receive an email from certification@reso.org with links to review and approve your results. The body of the email will contain a review link for each endorsement submitted by your Provider.
+You will receive an email from certification@reso.org with link(s) to review and approve your results. The body of the email will contain a review link for the endorsements that were submitted by your Provider.
```
Comment (Jad212s) [inline docs/recipients.md]: Should be "link" singular with the bundled configuration.

File: docs/recipients.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 # Notifications
-You will receive an email from certification@reso.org with links to review and approve your results. The body of the email will contain a review link for each endorsement submitted by your Provider.
+You will receive an email from certification@reso.org with link(s) to review and approve your results. The body of the email will contain a review link for the endorsements that were submitted by your Provider.
 
 Select the review and approve results link below the endorsement you would like to review.
```
Comment (Jad212s) [inline docs/recipients.md:4]: Rewrite as "Select the **REVIEW AND APPROVE RESULTS** link located below the endorsement(s) to be taken to the certification summary page."

**Review by gsaxRESO — APPROVED** [review]
I read through the Providers and Recipients guides and suggested several light changes that appear to have been accepted by Jason. I believe that we're in good shape here.

**Comment by gsaxRESO** [comment]
GitHub protocol and terminology is still fairly new to me, so please forgive any indiscretions during my editing, review and approval. I'm learning as I go and researching GitHub more, since we intend to use it more as a team at RESO.

**Pull request merged**